ecs.php

```diff
@@ -377,26 +373,13 @@
             ReferenceThrowableOnlySniff::class,
             // The @param, @return, @var and inline @var annotations should keep standard format
             ParamReturnAndVarTagMalformsFixer::class,
-            // Takes `@var` annotation of non-mixed types and adjusts accordingly the property signature.
-            PhpdocToPropertyTypeFixer::class,
-            // Takes `@param` annotations of non-mixed types and adjusts accordingly the function signature.
-            PhpdocToParamTypeFixer::class,
-            // Takes `@return` annotation of non-mixed types and adjusts accordingly the function signature.
-            PhpdocToReturnTypeFixer::class,
-            // Promote constructor properties
-            // @see https://github.com/FriendsOfPHP/PHP-CS-Fixer/issues/5956
-            RequireConstructorPropertyPromotionSniff::class,
 
             // switch -> match
             // @see https://github.com/FriendsOfPHP/PHP-CS-Fixer/issues/5894
 
             // Require \Stringable interface in classes implementing __toString() method
             // > it may probably be a phpstan rule, more than cs rule - since it needs a class hierarchy to solve this
             // @see https://github.com/FriendsOfPHP/PHP-CS-Fixer/issues/6235
-            PropertyTypeHintSniff::class,
-
-            ParameterTypeHintSniff::class,
```

Comment:
Why are we removing those two?

Reply:
And if we rly don't want them, maybe we should also remove https://github.com/lmc-eu/php-coding-standard/blob/af892d76815b3197529bec671ffe4e1df5f5bf59/ecs.php#L521?